core/kazoo_media/src/kz_media_util.erl

```diff
@@ -370,7 +371,8 @@ recording_url(CallId, Data) ->
 
 -spec max_recording_time_limit() -> ?SECONDS_IN_HOUR.
 max_recording_time_limit() ->
-    kapps_config:get_integer(?CONFIG_CAT, <<"max_recording_time_limit">>, ?SECONDS_IN_HOUR).
+    erlang:max(kapps_config:get_integer(?CONFIG_CAT, <<"max_recording_time_limit">>, ?DEFAULT_MAX_RECORDING_LIMIT)
+              ,?DEFAULT_MAX_RECORDING_LIMIT).
```

Comment:
Drop `).` to next line to align with the opening `(` from erlang:max/2

Reply:
I also am not sure about this erlang:max/2 usage. If I knowingly limit recordings to 1 hour in the system config, this will over-rule my config and use 3 hours, no?

Reply:
@jamesaimonetti Your concern is valid that this code will overwrite any number lower than the threshold, even if it's knowingly set.
@k-anderson can you help to weigh in what should be ceiling/floor of max_recording_time_limit. I explained in the PR description about why it's implemented such way. Thanks!